data/tlg0006/tlg006/tlg0006.tlg006.perseus-grc2.xml

```diff
@@ -0,0 +1,2532 @@
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"  encoding='UTF-8'?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+   <teiHeader xml:lang="eng">
+      <fileDesc>
+         <titleStmt>
+            <title type="work" n="Andr.">Andromache</title>
+            <title type="sub">Machine readable text</title>
+            <author n="Eur.">Euripides</author>
+            <editor role="editor" n="Murray">Gilbert Murray</editor>
+            <sponsor>Perseus Project, Tufts University</sponsor>
+            <principal>Gregory Crane</principal>
+            <respStmt>
+               <resp>Prepared under the supervision of</resp>
+               <name>Gregory Crane</name>
+               <name>Alison Babeu</name>
+               <name>Lisa Cerrato</name>
+            </respStmt>
+         </titleStmt>
+         <extent>About 100Kb</extent>
+         <publicationStmt>
+            <publisher>Trustees of Tufts University</publisher>
+            <pubPlace>Medford, MA</pubPlace>
+         </publicationStmt>
+         <sourceDesc default="false">
+            <bibl default="false">
+               <author>Euripides</author>
+               <title>Euripidis Fabulae, Vol 1</title>
+               <editor role="editor">Gilbert Murray</editor>
+               <pubPlace>Oxford</pubPlace>
+               <publisher>Clarendon Press, Oxford</publisher>
+               <date>1902</date>
+            </bibl>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <refsDecl n="CTS">
+            <cRefPattern n="line" matchPattern="(.+)" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:sp/tei:l[@n=ʼ$1ʼ])">
+               <p>This pointer pattern extracts line</p>
+            </cRefPattern>
+            
+         </refsDecl>
+         
+      </encodingDesc>
+   </teiHeader>
+   <text xml:lang="grc">
+      <body>
+         <head>Andromache</head>
+         <note type="title" place="inline" anchored="true">Ἀνδρομάχη</note>
+         <milestone n="1" unit="card"/>
+          <div type="edition" xml:lang="grc" n="urn:cts:greekLit:tlg0085.tlg001.perseus-grc2">
+        <div type="textpart" n="episode" org="uniform" sample="complete" part="N">
+            <sp>
```

Comment:
add ` <l resp="perseus" style="hidden" n="0"/>` in order to see the first speaker label